apps/mdn/mdn-aws/k8s/celery.beat.deploy.yaml.j2

```diff
@@ -1,6 +1,8 @@
 {% include 'kuma.base.deploy.yaml.j2' %}
           command:
-            - ./manage.py
+            - newrelic-admin
```

Comment:
Let's use the full path ``/usr/local/bin/newrelic-admin`` to avoid ``PATH`` issues.